velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -335,11 +362,163 @@ TEST_F(ParquetTpchTest, q6) {
   const auto& duckDBRounded =
       fmt::format("select round(revenue, 2) from ({})", duckDbSql);
 
-  auto task = assertQuery(params, filePath, sourcePlanNodeId, duckDBRounded);
+  auto task = assertQuery(params, {filePath}, {sourcePlanNodeId}, duckDBRounded);
 
   const auto& stats = task->taskStats();
   // There should be two pipelines
   ASSERT_EQ(2, stats.pipelineStats.size());
   // We used the default of 10 splits
   ASSERT_EQ(10, stats.numFinishedSplits);
 }
+
+TEST_F(ParquetTpchTest, q18) {
+  static std::string rawSQL = R"(
+    SELECT
+        c_name,
+        c_custkey,
+        o_orderkey,
+        o_orderdate,
+        o_totalprice,
+        sum(l_quantity) AS quantity
```

Comment:
I had to add the query as string here because I wasn't sure how to pick the last column while rounding off double values. and whatever I tried didn't work. Any suggestions here are welcome.

Reply:
I did try this and didn't work.

Reply:
You cannot use a raw string literal. The following works.
```
  auto duckDbSql = duckDb_->getTpchQuery(18);
  const auto& duckDBRounded = fmt::format(
        "select c_name, c_custkey, o_orderkey, o_orderdate,"
        "round(o_totalprice, 2) as o_totalprice, \"sum(l_quantity)\" from ({})"
        , duckDbSql);
```

Reply:
This worked 👍🏾 thanks, although I would like to have a string literal in there - I think it makes the SQL statement more readable.